pylint/checkers/imports.py

```diff
@@ -39,7 +39,7 @@
 
 # The dictionary with Any should actually be a _ImportTree again
 # but mypy doesn't support recursive types yet
-_ImportTree = Dict[str, List[Dict[str, Any] | List[str]]]
+_ImportTree = Dict[str, Union[List[Dict[str, Any], List[str]]]]
 
```

Comment:
I don't understand this change, is'nt the ``|`` operator better ?

Reply:
The operator isn't supported for runtime executed statements. The future import allows to use them in typing, but aliases are executed so we need to use the typing imports directly rather than the more modern notation.